pom.xml

```diff
@@ -142,7 +142,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>21.0</version>
+            <version>23.0</version>
```

Comment:
https://search.maven.org/artifact/com.google.guava/guava/27.0.1-jre/bundle

So, why 23.0 ?

Reply:
Ok, 23 is good enough anyway